pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
@davidopp --can we separate the problem and ask if the pod spec should allow mutation to any toleration, or if we should allow for example only new tolerations to be added to an existing pod spec.